newbank/server/commands/MoveMoney.java

```diff
@@ -10,10 +11,11 @@
 public class MoveMoney implements Command {
 
     private final CustomerManager theCustomerManager = CustomerManager.getInstance();
+    private final Authenticator theAuthenticator = Authenticator.getInstance();
 
     public CommandResponse process(ArrayList<String> argsList) throws CommandException {
 
-        if (argsList.size() != 4) {
+        if (argsList.size() != 4 && argsList.size() != 5) {
```

Comment:
(for my own curiosity)

If the argsList.size() is neither 4 nor 5, it throws up the given exception. I assume you have changed the value to 5 because of the new additions,  but I am curious as to why "argsList.size() != 5" would not suffice on its own.

Reply:
I have added both so that if someone has not setup 2FA they can still move money without the need to enter a 2FA code.